### DISCUSSION TITLE
SCIM in casdoor

### DISCUSSION THREAD

**Discussion by majodu** [casdoor/casdoor#2275] [General]
Is there any plans to add a scim applications like you can do in okta? The goal is to use casdoor for all identity management of users and have my application associate a users with my applications data? This is my very loose understanding of SCIM. Has anyone done this?

**Comment by hsluoyz** [comment]
@majodu SCIM is supported by Casdoor starting from v1.430.0: https://github.com/casdoor/casdoor/releases/tag/v1.430.0
